### PR TITLE
Add more Cassandra log

### PR DESCRIPTION
**Pull request by wxing1292**
**What changed?**
* Add more Cassandra log

**Why?**
N/A

**How did you test it?**
N/A

**Potential risks**
N/A

### CODE REVIEW

**Review by samarabbas — APPROVED** [review]

File: common/persistence/cassandra/cassandraHistoryPersistence.go [hunk 1]
```diff
@@ -122,6 +123,9 @@ func (h *cassandraHistoryV2Persistence) AppendHistoryNodes(
 		batch.Query(v2templateUpsertData,
 			branchInfo.TreeId, branchInfo.BranchId, request.NodeID, request.TransactionID, request.Events.Data, request.Events.EncodingType.String())
 		err = h.session.ExecuteBatch(batch)
+		h.logger.Error("####### BATCH QUERY ERROR")
+		h.logger.Error(prettyPrint(batch.Entries), tag.Error(err))
```
Comment (samarabbas) [inline common/persistence/cassandra/cassandraHistoryPersistence.go:127]: Can you also log workflow and run ID?

**Comment by mfateev** [comment]
Is any way to log the response before it is processed by the gosql client?

File: common/persistence/cassandra/cassandraHistoryPersistence.go (hunk 1 above)
Comment (wxing1292) [reply common/persistence/cassandra/cassandraHistoryPersistence.go]: workflow ID & run ID is not accessible from here

**Comment by wxing1292** [comment]
> Is any way to log the response before it is processed by the gosql client?

the PR is updated with a session tracer, not sure whether that will work

**Review by Ardagan — APPROVED** [review]

**Pull request merged**